samtranslator/plugins/api/implicit_api_plugin.py

```diff
@@ -39,6 +39,8 @@ def __init__(self):
 
         self.implicit_api_logical_id = GeneratedLogicalId.implicit_api()
         self.existing_implicit_api_resource = None
+        self.conditions = set()
+        self.implicit_api_condition = 'SamImplicitApiCondition'
```

Comment:
This is the name of the condition that SAM adds to the template. Is this fine, or should the name be something else?

Reply:
Yes, it is called "ServerlessRestApi". Will change.